pycbc/filter/resample.py

```diff
@@ -342,5 +389,5 @@ def interpolate_complex_frequency(series, delta_f, zeros_offset=0, side='right')
 
     return out_series
 
-__all__ = ['resample_to_delta_t', 'highpass', 'interpolate_complex_frequency', 'highpass_fir', 'lowpass_fir', 'notch_fir', 'fir_zero_filter']
+__all__ = ['resample_to_delta_t', 'highpass', 'lowpass', 'interpolate_complex_frequency', 'highpass_fir', 'lowpass_fir', 'notch_fir', 'fir_zero_filter']
```

Comment:
It seems Code Climate was complaining about this line being too long. Make sure it does not go beyond 79 characters.